docs/logagent/changelog.md

```diff
@@ -1,6 +1,39 @@
 title: Sematext Logagent Changelog
 description: Features and improvements changelog for Sematext Logagent
 
+## Breaking changes in Logagent v3.x
+
+### Geo IP lookups
+
+Logagent v2.x handles Geo IP lookups in the parser module. Adjusting fields, containing IP addresses, required changes in patterns.yml 
+
+
+Logagent v3.x has a new [geoip plugin](output-filter-geoip.md) the list of fields for lookups are specified via environment variable or command line flag. 
+
+Note Logagent v3.x requires the setting for GEOIP_FIELDS (default value `client_ip`). It ignores the settings in patterns.yml. If you used custom field names make sure you set `GEOIP_FIELDS` environment variable or `--geoipFields command line flag. 
+
+*Backward compatible environment variables:*
+
+  - `GEOIP_ENABLED=<true|false>` - enable Geo IP lookups 
+  - `MAXMIND_DB_DIR=/tmp/` - set DB directory
+
+*Backward compatible command line flags:*
+
+  - `--geoipEnabled <true|false>`
+
+*New environment variable:*
+
+ - `GEOIP_FIELDS="client_ip,remote_address"` - list of fields
```

Comment:
Line 26 has inconsistent indenting. Look at the indent of all the other bullet points.